test/clj/rems/test_form_validation.clj

```diff
@@ -356,6 +356,37 @@
                                         :field/optional true
                                         :field/visible  true
                                         :field/value    "invalid.email"}]))))
+  
+    (testing "error: field input invalid phone number"
+      (is (= [{:type :t.form.validation/invalid-phone-number :field-id "2"}
+              {:type :t.form.validation/invalid-phone-number :field-id "3"}
+              {:type :t.form.validation/invalid-phone-number :field-id "4"}
+              {:type :t.form.validation/invalid-phone-number :field-id "5"}]
+             (validate-fields-for-draft [{:field/id       "1"
+                                          :field/type     :phone-number
+                                          :field/optional false
+                                          :field/visible  true
+                                          :field/value    ""}
+                                         {:field/id       "2"
```

Comment:
The `:field/id` could be used to document the test case. It's always good to try to show the purpose of each case.

Reply:
This could be so `:field/id "prefix 0 is not allowed"`

Reply:
sounds like a really good idea!